arcor2_AREditor/Assets/TABLET/Scripts/GUI/ActionParametersMenu.cs

```diff
@@ -65,6 +65,9 @@ public class ActionParametersMenu : RightMenu<ActionParametersMenu>
         if (isValueValid && currentAction.Parameters.TryGetValue(parameterId, out Parameter actionParameter)) {           
             try {
                 if (JsonConvert.SerializeObject(newValue) != actionParameter.Value) {
+                    if (newValue == null) {
+                        actionParameter.SetValue(actionParameter.ParameterMetadata.GetDefaultValue());
```

Comment:
Just a tip for simpler code - can be rewritten as `newValue ??= actionParameter...`

Reply:
Not in this case as the value is not assigned to the "newValue" variable.

Reply:
I see, sorry :)